addons/html_editor/static/src/core/selection_plugin.js

```diff
@@ -690,6 +696,53 @@ export class SelectionPlugin extends Plugin {
         );
     }
 
+    /**
+     * @deprecated use `getTargetedNodes` instead.
+     *
+     * Returns an array containing all the nodes fully contained in the selection.
+     *
+     * @returns {Node[]}
+     */
+    getSelectedNodes() {
+        return this.getTraversedNodes().filter(this.areNodeContentsFullySelected.bind(this));
+    }
```

Comment:
I guess you should duplicate the tests as well because we don't want a future bug fix to break those. Even though we don't use them anymore, other people might, that's the whole point of the stable policy. (Even if I agree it's a bit ridiculous in our case but you never know, in fact we know of one company who builds on top of the editor!)

Reply:
@dmo-odoo I un-removed the `getTraversedNodes` and `getSelectedNodes` methods and adapted code in two new post-review commits so they in fact behave exactly as they used to. Let me know if this is good and I should squash my commits, or not ;-)